src/pages/DelegationCreatePage/DelegationCreate.page.tsx

```diff
@@ -0,0 +1,82 @@
+import { PageContainer, SectionContainer } from '@/components/layout/containers'
+import { useNavigate } from '@/router'
+import { Stack } from '@mui/material'
+import React, { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { DelegationCreateCards } from './components/DelegationCreateCards'
+import { StepActions } from '@/components/shared/StepActions'
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward'
+import { match } from 'ts-pattern'
+import { DelegationCreateForm } from './components/DelegationCreateForm'
+
+/***
+ * It shows the cards component to choose delegation kind or the form component based on the state KIND or FORM
+ */
+export const DelegationCreatePage: React.FC = () => {
+  const { t } = useTranslation('party')
+  const navigate = useNavigate()
+  const [delegationKind, setDelegationKind] = useState<'CONSUME' | 'PROVIDE' | ''>('')
```

Comment:
The absence of value should be represented with `null`. The `DelegationKind` type should be imported from the api types